### ISSUE TITLE
can no longer see Gistpad in sidebar

### ISSUE THREAD

**Issue by kidbrax** [vsls-contrib/gistpad#154]
**Describe the bug**
The gistpad is no longer availalbe as an option in the sidebar. 

**To Reproduce**
open vscode and right click on sidebar to choose visible panels. Gistpad will not be listed.

**Expected behavior**
Expect to be able to choose gistpad as a visible panel

**Screenshots**

**Desktop (please complete the following information):**
 - VS Code Version [e.g. v1.45.1]
 - GistPad Version [e.g. v0.0.68]

**Additional context**
I see Gistpad has some integration with Codetour in the changelog. And I now see Codetour as a panel but not Gistpad. I don't want Codetour installed either.

**Comment by lostintangent** [comment]
@kidbrax Hey! We made a recent change that removed the `GistPad` view from the `Explorer` tab, since `GistPad` also has its own tab (which I've found most users use). Does using the activity tab work for you or did you prefer having the `Explorer` view? The reason I removed it is because the `Explorer` view already gets pretty cluttered, and so I wanted to minimize that as much as possible.

![image](https://user-images.githubusercontent.com/116461/83679590-7f6fd600-a594-11ea-91ad-1e331a4e6e42.png)
